.travis.yml

```diff
@@ -4,17 +4,14 @@ node_js:
 env:
   global:
     - CC_TEST_REPORTER_ID=0be009d55f19435b5d362efac7afcdc532945bbc13527c285135c5e7b84f4e3c
-cache:
-  yarn: true
-  directories:
-  - node_modules
+cache: npm
 before_script:
 - cd webnode/
 - curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
 - chmod +x ./cc-test-reporter
 - ./cc-test-reporter before-build
 script:
-- yarn && yarn test:coverage
+- npm run test:coverage
```

Comment:
You still need to install dependencies with `npm ci`. Might make sense to create an `install` step in travis so it can cache the install steps.

Reply:
Nvm, I see you updated it. Didn't we want to use `npm ci`? Also still might make sense to do that in the `install` step instead of `script` so travis can cache it.

Reply:
travis runs npm ci instead of npm install: https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#npm-ci-support